Dockerfile

```
@@ -1,6 +1,4 @@
-FROM python:3.9
-
-RUN apt update && apt install -y git
+FROM python:3.9-slim-buster
```

```suggestion
FROM --platform=linux/amd64 python:3.9-slim-buster
```
lets do this to be nice to people on M1 macs that are amd64-challenged